modules/zip/package.json

```diff
@@ -25,6 +25,9 @@
     "dist",
     "README.md"
   ],
+  "browser": {
+    "fs": false
```

Comment:
I try to avoid having such node dependencies in more than the core modules. They tend to cause build problems for applications. This is a pretty good reason to consider moving just these file handlers to loader-utils, which already handles this problem

I don't want to ask you make multiple changes so fine for now.

Reply:
Yes. I thought about moving these classes to some core module. I will do it in a separate PR